### DISCUSSION TITLE
Is it possible to update mbed-ce/mbed-os from the original mbed-os?

### DISCUSSION THREAD

**Discussion by jancotipsarevic** [mbed-ce/mbed-os#124] [Q&A] [answered]
I am interested in new targets that have been recently included in the original mbed-os, how is the proper way to use them in mbed-ce?
How is the strategy updating mbed-ce with original mbed-os updates?

Thank you very much

**Comment by JohnK1987** [comment, marked as answer]
Hello, 

can you be more specific about "new targets"?
A merge is done from time to time. Look to the last one - https://github.com/mbed-ce/mbed-os/pull/117

BR, Jan
